backend/handlers/institution_parent_request_collection_handler.py

```diff
@@ -125,4 +127,9 @@ def main_operations(request, requested_inst_key, child_institution, user, host):
 
         request = main_operations(request, requested_inst_key, child_institution, user, host)
 
+        requested_inst = requested_inst_key.get()
```

Comment:
The sending of notification should be in a queue of tasks, not to cause a bottleneck in the client's request.